tests/runtests.py

```diff
@@ -396,8 +413,8 @@ def paired_tests(paired_test, options, test_labels, parallel):
              'LiveServerTestCase) is expected to run from. The default value '
              'is localhost:8081-8179.')
     parser.add_argument(
-        '--selenium', action='store_true', dest='selenium', default=False,
-        help='Run only the Selenium tests (equivalent to "--tag selenium").')
+        '--selenium', dest='selenium', action=ActionSelenium,
+        help='A comma-separated list of browsers to run only the Selenium tests against.')
```

Comment:
I think we should chop off "only". `--selenium` doesn't ensure running of selenium tests only. For example, `./runtests.py --tag=slow --selenium=firefox` will run both "slow" and "selenium" tests. I had written the logic to "append" selenium to the list of tags (not to override it).

Also we could add ` metavar='BROWSERS'` here. As `ArgumentError` is being raised on providing an invalid browser spec (showing the whole usage manual of `runtests.py`), it would be helpful and make sense.